client/gui/SDL_Extensions.h

```diff
@@ -152,6 +152,60 @@ struct ColorPutter
 
 typedef void (*BlitterWithRotationVal)(SDL_Surface *src,SDL_Rect srcRect, SDL_Surface * dst, SDL_Rect dstRect, ui8 rotation);
 
+class ColorShifter
+{
+public:
+	virtual SDL_Color shiftColor(SDL_Color clr) const = 0;
+};
+
+class ColorShifterLightBlue : public ColorShifter
+{
+public:
+	static ColorShifterLightBlue * create()
+	{
+		std::unique_ptr<ColorShifterLightBlue> shifter(new ColorShifterLightBlue());

```

Comment:
`shifter` should be static

Reply:
I didn't plan to make the "shifter" to be Singeleton, so here it only to create a new one.

Reply:
If "shifter" is not singleton, then you need to return smart pointer from this function or just remove it.

Reply:
Removed the functions.